File: Multiples_of_3_or_5/Solution.java
public class Solution {

    public static int solution(int number) {
        int res = 0;
        for (int i = 0; i < number; i++) {
            res += (i % 3 == 0 || i % 5 == 0) ? i : 0;
        }
        return res;
    }

    public static void main(String[] args) {
        System.out.println(solution(10));
    }
}

File: Multiples_of_3_or_5/Solution.c
int solution(number) 
{
    int i;
    int res;

    i = 0;
    res = 0;
    while (i < number)
    {
        if ((i % 3 == 0) || (i % 5 == 0))
            res += i;
        i++;
    }
    return res;
}

int main(void) 
{
    solution(10);
    return (0);
}
